homeassistant/components/gotify/strings.json

```diff
@@ -0,0 +1,22 @@
+{
+  "title": "gotify",
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "token": "[%key:common::config_flow::data::api_token%]"
+        }
+      }
+    },
+    "error": {
+      "invalid_host": "[%key:common::config_flow::error::invalid_host%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
Can we set a config flow unique id?
https://developers.home-assistant.io/docs/config_entries_config_flow_handler#unique-ids
If this is not possible then delete the unused already_configured string.

Reply:
I don't think a unique id is suitable in this case having read through the linked documentation. I will remove this string.